dts/arm/ene/kb1200.dtsi

```diff
@@ -42,6 +42,22 @@
 				reg = <0x10000000 0x40000>;
 			};
 		};
+
+		pmu: pmu@40010000 {
+			reg = <0x40010000 0x100>;
+		};
+
+		gcfg: gcfg@40000000 {
+			reg = <0x40000000 0x100>;
+		};
+
+		pinctrl: pin-controller@50000000 {
+			compatible = "ene,kb1200-pinctrl";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			reg = <0x50000000 0x100>;
+			status = "okay";
+		};
```

Comment:
this addition has problems:

1. makes commits non-bisectable (you're adding soc code depending on pmu/gcfg nodelabels before
2. you're adding pinctrl here, plus other stuff which is not explained on how it relates to pinctrl (pmu/gcfg)